node/src/gql/query/mentor-client-data.ts

```diff
@@ -250,8 +259,8 @@ async function getTopicQuestions(
       return a.name.localeCompare(b.name) * -1;
     });
     sQuestions.sort((a, b) => {
-      const qa = questions.find((q) => `${q._id}` === `${a.question}`);
-      const qb = questions.find((q) => `${q._id}` === `${b.question}`);
+      const qa = questionDict[`${a.question}`];
```

Comment:
why bother with the string manipulation here?

Reply:
question here is a mongoose ObjectId type, have to stringify it to compare to strings